superset-frontend/package.json

```diff
@@ -51,8 +51,9 @@
     "dev": "webpack --mode=development --color --watch",
     "dev-server": "cross-env NODE_ENV=development BABEL_ENV=development node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --mode=development",
     "format": "npm run _prettier -- --write",
-    "lint": "npm run _lint && npm run type",
-    "lint-fix": "npm run _lint -- --fix && npm run type",
+    "lint": "npm run _lint -- .",
```

Comment:
using base command, but maintaining original behavior, even though it's a `_` private command. Also removing `tsc` command to run here. It's a different linter and type of checks altogether, let's break this apart.